go.mod

```diff
@@ -132,7 +132,7 @@ require (
 	istio.io/api v1.19.2-0.20231011000955-f3015ebb5bd4 // indirect
 	istio.io/client-go v1.19.3 // indirect
 	k8s.io/apiextensions-apiserver v0.28.3 // indirect
-	k8s.io/autoscaler v0.0.0-20190805135949-100e91ba756e // indirect
+	k8s.io/autoscaler/vertical-pod-autoscaler v1.0.0 // indirect
```

Comment:
https://github.com/kubernetes/autoscaler/releases/tag/vertical-pod-autoscaler-1.0.0 was tagged as Pre-release.
Would it be possible to directly bump up to v1.1.0 without causing issues with the Gardener dependency?

Reply:
@renormalize If you observe, [all releases](https://github.com/kubernetes/autoscaler/releases?q=vertical-pod-autoscaler&expanded=true) of `k8s.io/autoscaler/vertical-pod-autoscaler` other than the latest version are marked as pre-release. I'm guessing this might have something to do with the fact that one single repo https://github.com/kubernetes/autoscaler has multiple components which have their own releases and tags, so the tagging of these releases may not be accurate, and more of a workaround, hence all previous releases are marked as pre-release, which doesn't really make sense if you see it practically. Also, if you observe, the other components like `Cluster Autoscaler` and `Addon Resizer` don't have any release tags like `latest` or `pre-release` at all.

Also, since this is an indirect dependency for etcd-druid, we need not worry about its exact version, as long as it corresponds to the direct dependencies that are vendored, like gardener/gardener v1.86.0, which is `k8s.io/autoscaler/vertical-pod-autoscaler:v1.0.0`, as you can see [here](https://github.com/gardener/gardener/blob/cee120166b7b8a857dea90fc0217d86c2e41850f/go.mod#L54).

Reply:
Thanks for the explanation.